packages/st-lukesmith/st-lukesmith.pacscript

```diff
@@ -1,12 +1,15 @@
 name="st-lukesmith"
 pkgname="st"
-version="0.16.1"
 url="https://github.com/LukeSmithxyz/st.git"
 license="MIT"
 build_depends="make libharfbuzz-bin libxft-dev libpango1.0-dev"
 depends="fontconfig"
 gives="st"
 
+pkgver () {
+  git ls-remote https://github.com/LukeSmithxyz/st.git master | sed "s/refs\/heads\/.*//" # Returns the latest commit hash from remote
+}
+version="${pkgver}"
```

Comment:
Why do you have a `pkgver()` function in a release package?

Reply:
Oh this is a different pacscript...


Why are you making 1 PR for two different pacscripts...